lib/liquid/tags/echo.rb

```diff
@@ -20,6 +22,12 @@ def initialize(tag_name, markup, parse_context)
     def render(context)
       @variable.render_to_output_buffer(context, +'')
     end
+
+    class ParseTreeVisitor < Liquid::ParseTreeVisitor
+      def children
+        Liquid::Variable::ParseTreeVisitor.new(@node.variable, @callbacks).children
```

Comment:
I don't think there is a `@callbacks` in this context.  Also, this seems quite different from other tags which just return an array rather than using `ParseTreeVisitor.new` directly.

The assign tag is another tag that embeds a variable, so I would expect it to be similar to it

```suggestion
        [@node.variable]
```

Reply:
That is much better indeed.

Reply:
Done.